.gitignore

```diff
@@ -0,0 +1,5 @@
+.DS_Store
+.tox/*
+site/*
```

Comment:
Please, let's not add `.tox` and `site` to `.gitignore`. I am quite fond of the idea of being able to restore a Git checkout to its pristine condition with `git clean -fd`. :)

Reply:
Aha, I understand!